astropy/utils/tests/test_data.py

```diff
@@ -903,6 +904,14 @@ def test_find_invalid():
         )
 
 
+@pytest.mark.parametrize(("package"), [None, "astropy", "numpy"])
```

Comment:
Nitpick: Don't think parenthesis is needed when there is only one argument.
```suggestion
@pytest.mark.parametrize("package", [None, "astropy", "numpy"])
```